infra/aws/terraform/kops-infra-ci/eks.tf

```diff
@@ -17,7 +17,7 @@ limitations under the License.
 module "eks" {
   providers = { aws = aws.kops-infra-ci }
   source    = "terraform-aws-modules/eks/aws"
-  version   = "20.8.3"
+  version   = "20.10.0"
```

Comment:
any reason not to go up to `2.20.0`?

Reply:
I don't think we need the latest version but happy to do it.

Reply:
For this cluster, there is no real need to switch to Karpenter right now. Probably something to think about in 2025.